cost/aws/elb/clb_unused/AWS_Delete_Unused_CLB.pt

```diff
@@ -134,7 +134,7 @@ script "js_aws_clb_filter_map", type: "javascript" do
     var tagKeyValue=""
     for(var j=0; j < tags.length; j++){
       tag = tags[j]
-      if(param_exclude_tags.indexOf(tag['tagKey']+'='+tag['tagValue']) !== -1){
+      if(((param_exclude_tags.join()+',').toLowerCase()).indexOf((tag['tagKey']+'='+tag['tagValue']+',').toLowerCase()) !== -1){
```

Comment:
This wouldn't work when the current tag is the last tag in the exclude tag list.